api/model.go

```diff
@@ -2122,3 +2210,43 @@ func sliceToMap(conditions []longhorn.Condition) map[string]longhorn.Condition {
 	}
 	return converted
 }
+
+func toObjectStoreResource(store *longhorn.ObjectStore, allocatedSize, occupiedSize int64) *ObjectStore {
+	return &ObjectStore{
+		Resource: client.Resource{
+			Id:   store.Name,
+			Type: "objectStore",
+		},
+		Name:          store.Name,
+		State:         store.Status.State,
+		Endpoints:     store.Status.Endpoints,
+		AllocatedSize: allocatedSize,
+		OccupiedSize:  occupiedSize,
```

Comment:
Should we use `Size` instead of `AllocatedSize` and `ActualSize` instead of `OccupiedSize` to be uniform with the volume resource? Introducing naming might make it a bit harder to trace which value they are related to